---
title: "The ViReflow pipeline enables user friendly large scale viral consensus genome reconstruction"
collection: publications
pubtype: paper
permalink: /publication/2022-03-24-vireflow
excerpt: 'The ViReflow pipeline enables user friendly large scale viral consensus genome reconstruction'
date: 2022-03-24
venue: 'Scientific Reports'
paperurl: 'https://doi.org/10.1038/s41598-022-09035-w'
citation: '<b>Moshiri N</b>, Fisch KM, Birmingham A, DeHoff P, Yeo GW, Jepsen K, Laurent LC, Knight R (2022). "The ViReflow pipeline enables user friendly large scale viral consensus genome reconstruction." <i>Scientific Reports</i>. 12:5077. <a href="https://doi.org/10.1038/s41598-022-09035-w" target="_blank">doi:10.1038/s41598-022-09035-w</a>'
---
Throughout the COVID-19 pandemic, massive sequencing and data sharing efforts enabled the real-time surveillance of novel SARS-CoV-2 strains throughout the world, the results of which provided public health officials with actionable information to prevent the spread of the virus. However, with great sequencing comes great computation, and while cloud computing platforms bring high-performance computing directly into the hands of all who seek it, optimal design and configuration of a cloud compute cluster requires significant system administration expertise. We developed ViReflow, a user-friendly viral consensus sequence reconstruction pipeline enabling rapid analysis of viral sequence datasets leveraging Amazon Web Services (AWS) cloud compute resources and the Reflow system. ViReflow was developed specifically in response to the COVID-19 pandemic, but it is general to any viral pathogen. Importantly, when utilized with sufficient compute resources, ViReflow can trim, map, call variants, and call consensus sequences from amplicon sequence data from 1000 SARS-CoV-2 samples at 1000X depth in < 10 min, with no user intervention. ViReflow’s simplicity, flexibility, and scalability make it an ideal tool for viral molecular epidemiological efforts.
