---
title: "Swapping Metagenomics Preprocessing Pipeline Components Offers Speed and Sensitivity Increases"
collection: publications
pubtype: paper
permalink: /publication/2022-03-16-metagenomics-preprocessing-pipeline
excerpt: 'Swapping Metagenomics Preprocessing Pipeline Components Offers Speed and Sensitivity Increases'
date: 2022-03-16
venue: 'mSystems'
paperurl: 'https://doi.org/10.1128/msystems.01378-21'
citation: 'Armstrong G, Martino C, Morris J, Khaleghi B, Kang J, Dereus J, Zhu Q, Roush D, McDonald D, Gonzalez A, Shaffer J, Carpenter C, Estaki M, Wandro S, Eilert S, Akel A, Eno J, Curewitz K, Swafford A, <b>Moshiri N</b>, Rosing T, Knight R (2022). "Swapping Metagenomics Preprocessing Pipeline Components Offers Speed and Sensitivity Increases." <i>mSystems</i>. e01378-21. <a href="https://doi.org/10.1128/msystems.01378-21" target="_blank">doi:10.1128/msystems.01378-21</a>'
---
Increasing data volumes on high-throughput sequencing instruments such as the NovaSeq 6000 leads to long computational bottlenecks for common metagenomics data preprocessing tasks such as adaptor and primer trimming and host removal. Here, we test whether faster recently developed computational tools (Fastp and Minimap2) can replace widely used choices (Atropos and Bowtie2), obtaining dramatic accelerations with additional sensitivity and minimal loss of specificity for these tasks. Furthermore, the taxonomic tables resulting from downstream processing provide biologically comparable results. However, we demonstrate that for taxonomic assignment, Bowtie2â€™s specificity is still required. We suggest that periodic reevaluation of pipeline components, together with improvements to standardized APIs to chain them together, will greatly enhance the efficiency of common bioinformatics tasks while also facilitating incorporation of further optimized steps running on GPUs, FPGAs, or other architectures. We also note that a detailed exploration of available algorithms and pipeline components is an important step that should be taken before optimization of less efficient algorithms on advanced or nonstandard hardware.

**IMPORTANCE** In shotgun metagenomics studies that seek to relate changes in microbial DNA across samples, processing the data on a computer often takes longer than obtaining the data from the sequencing instrument. Recently developed software packages that perform individual steps in the pipeline of data processing in principle offer speed advantages, but in practice they may contain pitfalls that prevent their use, for example, they may make approximations that introduce unacceptable errors in the data. Here, we show that differences in choices of these components can speed up overall data processing by 5-fold or more on the same hardware while maintaining a high degree of correctness, greatly reducing the time taken to interpret results. This is an important step for using the data in clinical settings, where the time taken to obtain the results may be critical for guiding treatment.
